root/static/scripts/guess-case/modes.js

```diff
@@ -17,8 +17,9 @@ const {isPrepBracketWord, isPrepBracketSingleWord} = require('./utils');
  * keschte  2005-04-17  added french lowercase characters
  * keschte  2005-06-14  added "tha" to be handled like "the"
  * warp     2011-02-01  added da, de, di, fe, fi, ina, inna
+ * dpmittal 2016-12-20  added turkish lowercase characters
  */
-const LOWER_CASE_WORDS = /^(a|an|and|as|at|but|by|da|de|di|fe|fi|for|in|ina|inna|n|nor|o|of|on|or|tha|the|to)$/;
+const LOWER_CASE_WORDS = /^(a|an|and|as|at|but|by|da|de|di|fe|fi|for|in|ina|inna|n|nor|o|of|on|or|tha|the|to|ve|ile|ya|veya|yahut|ki|mı|mi|mu|mü|mısın|musun|mudur|mıdır|midir|miyim|misin|misiniz|mısınız|müsün|müyüm|müsünüz)$/;
```

Comment:
The following harmonic variants of the question particles are missing: `mıyım`, `muyum`, `musunuz`. The first person plural is missing altogether: `miyiz`, `mıyız`, `muyuz`, `müyüz`.

Regarding `mudur|mıdır|midir`, `müdür` is again missing; however, that might be intentional, because there is a homophone noun (meaning _headmaster_). I can’t judge whether it makes sense to break consistency here, and we should probably ask someone who speaks Turkish properly.